.jules/sentinel.md

```
@@ -0,0 +1,5 @@
+
+## 2025-02-18 - Fix XXE Vulnerability in JUnit XML Parsing
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)